Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM public.ecr.aws/prima/elixir:1.11.2-2
+FROM public.ecr.aws/prima/elixir:1.13.0-1
```

Comment:
why?

Reply:
[Drone got angry](https://drone-1.prima.it/primait/bridge_ex/143/1/7) and I thought this is what it wanted

Reply:
Yeah I meant why the 1.13 version requirement? (the comment should have been on the mix.exs line I guess -.-")

Reply:
I reverted the `mix.exs` version to `1.11` but kept the Dockerfile version as discussed